base/collect_http.go

```diff
@@ -21,8 +21,8 @@ type errorRange struct {
 	Upper *int `json:"upper" yaml:"upper"`
 }
 
-// collectHTTPInputs contain the inputs to the metrics collection task to be executed.
-type collectHTTPInputs struct {
+// CollectHTTPInputs contain the inputs to the metrics collection task to be executed.
```

Comment:
I do not understand why this has been capitalized. I do not see its  use outside of the package. The only code changes in this PR are to the base package.

Reply:
The only reason for capitalizing it is show that its documentation shows up in go.pkg.dev, which in turn I can reference in the values.yaml file.
***
Considering we can use the GitHub permalinks instead, I also don't see the value in  capitalizing the type. Reverting ...

Reply:
I was going to say this seems like a good reason :)